<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_domain">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Do&amp;main</string>
       </property>
       <property name="buddy">
        <cstring>combo_domain</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_domain">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_item_id">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Item</string>
       </property>
       <property name="buddy">
        <cstring>text_name_part</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="text_name_part">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_item_type">
       <item>
        <property name="text">
         <string>Device</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Macro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Server</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_find_projects">
       <property name="text">
        <string>&amp;Find Projects</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="table_projects"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bottom_horizontal_layout">
       <item>
        <widget class="QLabel" name="label_status">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Projects with Items</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
